<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PhotoINF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="originImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>390</width>
      <height>390</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>540</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="targetImage">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>390</width>
      <height>390</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>390</width>
      <height>390</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Z003</family>
      <pointsize>36</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>PhotoINF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadImage">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Carregar imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="horizMirror">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Espelhamento
horizontal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertMirror">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Espelhamento
vertical</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toGreyscale">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tons de cinza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quantizerBtn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantização</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadImage">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetar</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgDimensionsLbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="numOfShadesSpin">
    <property name="geometry">
     <rect>
      <x>931</x>
      <y>280</y>
      <width>41</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
    </property>
    <property name="displayIntegerBase">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>280</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Num. de tons</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveImage">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Salvar imagem</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionasd">
   <property name="text">
    <string>asd</string>
   </property>
  </action>
  <action name="actionCarregar_imagem">
   <property name="text">
    <string>Carregar imagem</string>
   </property>
  </action>
  <action name="actionSalvar_imagem">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Salvar imagem</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
